### PR TITLE
Chats sync

### PR DESCRIPTION
**Pull request by shirokov-kirill**
Please check this branch from the previous semester. There was no PR for it

### CODE REVIEW

File: app-web-frontend/src/MainComponents/Chats/ChatsPage.js [hunk 1]
```diff
@@ -62,52 +63,83 @@ function ChatsPage() {
...
-          text: "Good morning! Fine, thank you, Radjesh!"
+          senderId: 0, // my userID
+          created: "11:10",
+          content: "Good morning! Fine, thank you, Radjesh!"
         }
       ]
     }
   ]
+
   let [position, setPosition] = useState(0);
```
Comment (e2e4b6b7) [inline app-web-frontend/src/MainComponents/Chats/ChatsPage.js:84]: nit: A bit controversial solution. Better approach would be to have 2 separate components, when we are in chat and when we are in chat list.

**Review by e2e4b6b7 — APPROVED** [review]
Remove CrossOrigin and it is ready to be pushed

File: app-backend/src/main/kotlin/org/example/appbackend/controller/MessagingController.kt [hunk 2]
```diff
@@ -8,6 +8,7 @@ import org.slf4j.LoggerFactory
 import org.springframework.web.bind.annotation.*
 
 @RestController
+@CrossOrigin(origins = ["http://localhost:3000"])
```
[the remark this answers is not in the document]
Comment (shirokov-kirill) [reply app-backend/src/main/kotlin/org/example/appbackend/controller/MessagingController.kt:11]: Yep, removed

File: app-web-frontend/src/MainComponents/Chats/ChatsPage.js (hunk 1 above)
Comment (shirokov-kirill) [reply app-web-frontend/src/MainComponents/Chats/ChatsPage.js:84]: Those already are separate components. Or do you mean the interface should be structured in another way?

**Pull request merged**